### ISSUE TITLE
Request - Clear Logo OSD

### ISSUE THREAD

**Issue by JFG1000** [SerpentDrago/skin.auramod#90]
![20210511_215452](https://user-images.githubusercontent.com/76907836/117888779-7a3dff80-b2aa-11eb-8612-0aedab0c79be.jpg)

Would it please be possible to implement a setting to allow us users to prevent the clear logo from appearing on the OSD like in this images, I've been told otherwise but as far as i can remember prior to the latest matrix branch update for me personally it's never appeared there before, only on the seeker bar. Many thanks

**Comment by SerpentDrago** [comment]
I can agree this would be a nice feature to have .  Some may want it on one or the other . 

But currently it works as it was org intended to work (show on both , with option to disable it on seekbar only ) and option to choose if its clearart or clearlogo.

The theory behind this was that users , if already displaying ALL that info on the screen would want even more info and graphics . but if they only had the seekbar up , it could get in the way . so a option was only ever added to disable it on seekbar for a nice clean look while seeking

**Comment by jojobrogess** [comment]
@SerpentDrago 
Looks like this wasn't `$LOCALIZE`'d :
https://github.com/SerpentDrago/skin.auramod/blob/002d53bcd883901795dab0ee9c392d50791c0a58/1080i/SkinSettings.xml#L1593


@JFG1000
> 
> 
> Would it please be possible to implement a setting to allow us users to prevent the clear logo from appearing on the OSD like in this images, I've been told otherwise but as far as i can remember prior to the latest matrix branch update for me personally it's never appeared there before, only on the seeker bar. Many thanks

It looks like you have Setting 4 toggled with the Clearart toggle on as well. 
If you take a look at the screenshot samples I've provided, maybe that will help you better understand how the settings operate.
If I'm completely off on this, at least it's here for the next guy

Setting:1 NO SETTING

![2021-05-11 15_27_11-Greenshot](https://user-images.githubusercontent.com/17692472/117894837-0ed83b80-b272-11eb-9e51-3052f2aae874.png)
![2021-05-11 15_58_52-Greenshot](https://user-images.githubusercontent.com/17692472/117894872-21527500-b272-11eb-8d9a-d52966d2ad8b.png)

Setting:2

![2021-05-11 16_20_06-Greenshot](https://user-images.githubusercontent.com/17692472/117896528-9b382d80-b275-11eb-9da2-62be6c4321a9.png)
![2021-05-11 16_20_19-Greenshot](https://user-images.githubusercontent.com/17692472/117896543-a12e0e80-b275-11eb-8aea-cf0957b65d35.png)


Setting:3

![2021-05-11 16_18_21-Greenshot](https://user-images.githubusercontent.com/17692472/117896077-9de65300-b274-11eb-9be2-fbabe73b2f11.png)
![2021-05-11 16_18_30-Greenshot](https://user-images.githubusercontent.com/17692472/117896078-9fb01680-b274-11eb-8245-fca3b6fd51a2.png)


Setting:4

![2021-05-11 16_04_41-Greenshot](https://user-images.githubusercontent.com/17692472/117895166-a6d62500-b272-11eb-9a7c-af5589eed16a.png)
![2021-05-11 15_58_52-Greenshot](https://user-images.githubusercontent.com/17692472/117895043-742c2c80-b272-11eb-8963-bbc29afb79a0.png)
**PRESS UP** or whatever your configuration button is to get to Info Controls
![2021-05-11 15_59_47-Greenshot](https://user-images.githubusercontent.com/17692472/117895022-6e364b80-b272-11eb-9937-66c057841c7b.png)

Setting:5

![2021-05-11 15_58_24-Greenshot](https://user-images.githubusercontent.com/17692472/117894962-4ba43280-b272-11eb-92b4-907a880e4597.png)
![2021-05-11 15_50_17-Greenshot](https://user-images.githubusercontent.com/17692472/117894423-25ca5e00-b271-11eb-8f7f-e2b9b735bb44.png)

Setting:6

![2021-05-11 15_53_00-Greenshot](https://user-images.githubusercontent.com/17692472/117894381-13e8bb00-b271-11eb-829b-a299815e1d62.png)
![2021-05-11 15_53_13-Greenshot](https://user-images.githubusercontent.com/17692472/117894382-14815180-b271-11eb-83a6-8731470b6aa8.png)


Setting's 1, 2, and 4 can all use the clearart/clearlogo toggle.
**EXAMPLE**


![2021-05-11 16_10_52-Greenshot](https://user-images.githubusercontent.com/17692472/117895635-a4c09600-b273-11eb-9cbb-b60066c03073.png)
![2021-05-11 16_11_04-Greenshot](https://user-images.githubusercontent.com/17692472/117895629-9ecab500-b273-11eb-9a34-8d161d0d2bcc.png)

**Comment by SerpentDrago** [comment]
Yes i went though it with him and well , i remember org having sky add all these options . 

he wants the ability to have clearlogo/clearart  ONLY on the "pause" screen , and NOT have it on the full info screen . 

that as far as i know and my years of experience with this skin can not be done . but those are great screen shots for others for sure !

**Comment by JFG1000** [comment]
It's not even on just on the pause screen, i press select on my remote to bring up the seeker bar and it's there, appreciate the advice but i do understand how it operates and went through it all with serpent before posting and played about with every setting i could
